src/config/config.py

```diff
@@ -45,6 +45,9 @@ def get_github_audit_report_key(self) -> str:
     def get_github_webhook_report_key(self) -> str:
         return self._get_env("GITHUB_WEBHOOK_REPORT_KEY")
 
+    def get_github_webhook_host_ignore_list(self) -> str:
+        return self._get_env("GITHUB_WEBHOOK_HOST_IGNORE_LIST")
```

Comment:
I would move the `.split(",")` here in the Config class and change the return type, given that the method name implies that this returns a list whereas it currently returns a string.